std/src/main/scala/algebra/std/list.scala

```diff
@@ -11,6 +11,14 @@ trait ListInstances {
   implicit def listMonoid[A] = new ListMonoid[A]
 }
 
+trait ListInstances1 {
+  implicit def listPartialOrder[A: PartialOrder] = new ListPartialOrder[A]
+}
+
+trait ListInstances2 {
```

Comment:
Are people meant to extend these traits directly? I would have expected `trait ListInstances1 extends ListInstances2` and `trait ListInstances extends ListInstances1` or something like that.